lldb/include/lldb/Target/Process.h

```diff
@@ -93,6 +93,9 @@ class ProcessProperties : public Properties {
   void SetDetachKeepsStopped(bool keep_stopped);
   bool GetWarningsOptimization() const;
   bool GetWarningsUnsupportedLanguage() const;
+#ifdef LLDB_ENABLE_SWIFT
+  bool GetWarningsToolchainMismatch() const;
```

Comment:
As a follow up patch, can we get rid of this function and the `PrintWarning.*` ones below by moving them into the language plugin?